ui/components/component-library/header-base/header-base.tsx

```diff
@@ -86,7 +85,6 @@ export const HeaderBase: React.FC<HeaderBaseProps> = ({
       )}
       {children && (
         <Box
-          className="mm-header-base__children"
           width={BLOCK_SIZES.FULL}
           style={getTitleStyles}
           {...childrenWrapperProps}
```

Comment:
Could make this a bit more robust by doing something like this
```suggestion
          {...childrenWrapperProps}
          style={{...getTitleStyles, ...childrenWrapperProps?.style}
          
```

Reply:
done